src/base/_bootstrap.scss

```diff
@@ -9,8 +9,5 @@ $icon-font-path: "#{$wb-core-path}/fonts/";
  * Copyright 2011-2015 Twitter, Inc.
  * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
  */
-/*!
- * GLYPHICONS Halflings for Twitter Bootstrap by GLYPHICONS.com | Licensed under http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I see that this pull request was closed without merging but I should caution about any future PRs that remove any license references from the code (since that is normally the condition that needs to be met to use the licensed code).